acs/acs.go

```diff
@@ -0,0 +1,92 @@
+package acs
+
+import (
+	"sync/atomic"
+
+	"github.com/DE-labtory/cleisthenes"
+	"github.com/DE-labtory/cleisthenes/bba"
+	"github.com/DE-labtory/cleisthenes/pb"
+	"github.com/DE-labtory/cleisthenes/rbc"
+)
+
+type request struct {
+	sender cleisthenes.Member
+	data   *pb.Message
+	err    chan error
+}
+
+type ACS struct {
+	// TODO : add Tracer
+
+	// rbcMap has rbc instances
+	rbcMap map[cleisthenes.Address]*rbc.RBC
+
+	// bbaMap has bba instances
+	bbaMap map[cleisthenes.Address]*bba.BBA
+
+	reqChan chan request
+
```

Comment:
If no comment above field, it looks good not to be spaced.

Reply:
Yep :) I will add.